admin/data/sql/migrations/026.do.create-group.sql

```diff
@@ -0,0 +1,18 @@
+CREATE TABLE [mtc_admin].[group]
```

Comment:
I strongly suggest we do NOT use reserved words for our table (and column) names.  Please rename, e.g. `groupName`